synth.py

```diff
@@ -23,11 +23,11 @@
 
 # Run the gapic generator
 gapic = gcp.GAPICGenerator()
-versions = ['v1beta1']
+versions = ['v1beta1', 'v1']
 for version in versions:
     library = gapic.node_library('containeranalysis', version,
             config_path=f"/google/devtools/containeranalysis/artman_containeranalysis_{version}.yaml")
-    s.copy(library, excludes=['package.json', 'README.md', 'src/index.js'])
+    s.copy(library, excludes=['package.json', 'README.md', 'src/index.js', 'test/gapic-v1beta1.js'])
```

Comment:
Why are we excluding the v1beta1 tests?

Reply:
The tests import the client incorrectly; seeing as all the sample tests fail without this old client (it seems as though the new API surface is quite different) I'm going to instead add the old export.